src/main/java/com/lesson52/dao/StudentRepository.java

```diff
@@ -0,0 +1,16 @@
+package com.lesson52.dao;
+
+import com.lesson52.entity.Student;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface StudentRepository extends JpaRepository<Student,Integer>
+{
+    List<Student> findByNameIgnoreCase(String name);
+    List<Student> findStudentBySurnameIgnoreCase(String surname);
+    List<Student> getStudentsByGroupIgnoreCaseTitle(String title);
+    List<Student> findStudentByUnPayIsFalse();
```

Comment:
Method should be named as `findStudentByUnPay` and `false` should be provided as a parameter